README.md

```diff
@@ -64,137 +87,159 @@ class { 'wsus_client':
 }
 ~~~
 
-### Enabling Status Server
+To automatically serve clients the correct updates, WSUS needs to know which updates the client has already installed. Clients can report this to a Windows Update status server as defined by the `WUStatusServer` policy, which must have the same value as the `WUServer` policy to be valid for automatic updates. For details, see the [Microsoft TechNet documentation][TechNet].
 
-The following will set registry keys for both `WUServer` and `WUStatusServer` to 'http://myserver:8530' as well as `UseWUServer` bit to `1` or `true`
+To configure Windows Update to use the same server for fetching updates and reporting client status, use the [`enable_status_server`][] parameter. For example, to configure a client to use `http://myserver:8530` for both updates and status reporting, declare this class:
```

Comment:
This enable_status_server link is broken. I don't immediately see why, partly because I don't use that style of links myself.